src/components/pages/cover/create/CreateNewCoverPage.jsx

```diff
@@ -0,0 +1,240 @@
+import FeeAndAmount from "@/components/pages/cover/create/FeeAndAmount";
+import GovernanceParameters from "@/components/pages/cover/create/GovernanceParameters";
+import PricingInput from "@/components/pages/cover/create/PricingInput";
+import { ResolutionSourcesInput } from "@/components/pages/cover/create/ResolutionSourcesInput";
+import { Checkbox } from "@/components/UI/atoms/checkbox";
+import { Container } from "@/components/UI/atoms/container";
+import { RegularInput } from "@/components/UI/atoms/input/regular-input";
+import { Label } from "@/components/UI/atoms/label";
+import { TagsInput } from "@/components/UI/atoms/tags-input";
+import DeleteIcon from "@/icons/delete-icon";
+import { classNames } from "@/utils/classnames";
+import React from "react";
+import { useState } from "react/cjs/react.development";
```

Comment:
This is causing the build to fail
```suggestion
import { useState } from "react";
```